products/appengine/terraform.yaml

```diff
@@ -87,6 +87,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         update_mask_fields:
           - "ssl_settings.certificate_id"
           - "ssl_settings.ssl_management_type"
+  UrlDispatchRule: !ruby/object:Overrides::Terraform::ResourceOverride
+    self_link: 'dispatch/update?app_id={{project}}'
+    id_format: "dispatch/update?app_id={{project}}"
```

Comment:
If we're setting a custom value, let's use `"{{project}}"`.

Reply:
updated.